test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -472,13 +476,17 @@ def setUp(self):
         if not PBSTestSuite.config_saved and self.use_cur_setup:
             _, path = tempfile.mkstemp(prefix="saved_test_setup",
                                        suffix=".json")
-            ret = self.server.save_configuration(path, 'w')
+            ret = self.server.save_configuration()
+            if not ret:
+                self.logger.error("Failed to save test setup")
+                raise Exception("Failed to save test setup")
+            ret = self.scheduler.save_configuration(path, 'w')
             if ret:
                 self.saved_file = path
-                PBSTestSuite.config_saved = True
             else:
                 self.logger.error("Failed to save test setup")
                 raise Exception("Failed to save test setup")
+            PBSTestSuite.config_saved = True
```

Comment:
I think `self.revert_schedulers` should be moved to the `else` block?

Reply:
Thanks for catching this.